assets/scss/_custom.scss

```
@@ -853,3 +853,13 @@ div.alert > em.javascript-required {
   min-height: 1.5em;
   margin: calc(max(4em, ( 8vh + 4em ) / 2)) 0 0.25em 0;
 }
+
+label
+{
+    font-weight: bold;
+
+    display: inline-block;
+
+    max-width: 100%;
+    margin-bottom: 5px;
+}
```

We need to retain some styles; for example:
```suggestion
.content__box
{
    margin: 0 0 20px;
    padding: 20px;
}

.content__box_lined
{
    box-shadow: inset 0 0 0 1px #326de6;
}

.content__box_fill
{
    color: #fff;
    background: #326de6;
}
```